deep-learning/_posts/2018-11-09-LearningImplicitBrainMRIManifolds.md

```diff
@@ -0,0 +1,107 @@
+author: "Jon Haitz Legarreta Gorroño"
+cite:
+    authors: "Camilo Bermudez and Andrew J. Plassard and Larry T. Davis and
+    Allen T. Newton and Susan M Resnick and Bennett A. Landman"
+    title:   "Learning Implicit Brain MRI Manifolds with Deep Learning"
+    venue:   "SPIE Medical Imaging 2018"
+pdf: "https://arxiv.org/pdf/1801.01847.pdf"
+---
+
+
+# Highlights
+Implicit manifold learning of brain MRI through two common image processing
+tasks:
+ - Unsupervised synthesis of T1-weighted brain MRI using a Generative
+ Adversarial Network (GAN) by learning from 528 examples of 2D axial slices
+ of brain MRI.
+- Autoencoder with skip connections for image denoising showing higher PSNR
+than FSL SUSAN after denoising.
+
+
+# Introduction
+
+The work assumes that a low-dimensional, non-linear manifold can effectively
+represent the variability found in brain anatomy.
+
+The mapping of a brain image into a manifold and vice versa can help construct
+priors that span the entire variability spectrum of a normal human brain.
+
+Authors hypothesize that the ideal manifold mapping would learn a naive
+representation using the entire image and from this manifold generate images
+that are of comparable wuality as the lerning set.
+
+Previous image synthesis works have produced "near-realistic" images, but
+"have not been validated for image quality by imaging experts" according to
+the authors of this paper.
+
+
+# Methods
+
+ ## Image synthesis
+- Generator and discriminator inspired from the Deep Convolutional Generative
+Adversarial Network ([DCGAN](https://arxiv.org/pdf/1511.06434.pdf))
+architecture.
+- Features "symmetric skip connections": adds the feature output from 2 conv
+layers to the corresponding deconv layer.
+
+![](/deep-learning/images/LearningImplicitBrainMRIManifolds/Architecture.png)
+
+ ## Image denoising
+- Training set: 528 noise vectors generated by adding Gaussian noise within a
+mask of the brain and skull to the 2D slices of the dataset.
+- Testing set: 100 new noise vectors.
+- Three different noisy images were generated for each subject with SNR of 1,
+10, and 100. Three separate instances of the proposed autoencoder network were
+trained for each level of noise.
+- Minimize the MSE.
+
+![](/deep-learning/images/LearningImplicitBrainMRIManifolds/DenoisingAutoencoderNetwork.png)
+
+## Data
+- 528 T1-weighted brain MPRAGE MRI images (single mid-line axial slice) of
+healthy subjects from the Baltimore Longitudinal Study of Aging
+([BLSA](https://www.blsa.nih.gov/)).
+- $$220 \times 172$$ size images with $$1$$ mm isotropic pixel resolution.
+- Affine-registered to MNIs-space and intensity-normalized.
+
+
+# Results
+
+## Image synthesis
+- Synthesized images were shown to be unique by performing a cross-correlation
+with the training set.
+- Real images with high correlation show several structural differences from
+the synthesized images, such as ventricle size or gyral patterns, suggestingcomparativ

```

Comment:
comparative*

Reply:
Done in 17a2dcd. Thanks for the review !